Easydict/App/Localizable.xcstrings

```diff
@@ -1970,6 +1970,23 @@
         }
       }
     },
+    "one_api" : {
+      "comment" : "The name of One API Translate",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "One API Translate"
```

Comment:
```suggestion
            "value" : "Custom OpenAI Translate"
```
How about renaming it to `Custom OpenAI Translate`?

`One API Translate` is not clear, I want user to know it's a general OpenAI service.

Reply:
ok